src/components/accounts/account-item.html

```diff
@@ -1,4 +1,4 @@
-<a class="accounts__list-item mdc-list-item" href="#">
+<a class="accounts__list-item mdc-list-item" account-id="<%= account.id %>" href="#">
```

Comment:
you can not introduce custom properties -- but use can use `data-*` properties -- so it will be like `data-id`. And the you will be able to reference this property via `el.dataset.id`